SU2_CFD/src/solvers/CEulerSolver.cpp

```diff
@@ -13408,3 +13897,246 @@ void CEulerSolver::GatherInOutAverageValues(CConfig *config, CGeometry *geometry
     }
   }
 }
+
+void CEulerSolver::ComputeTurboPerformance(CConfig *config, CGeometry *geometry) {
```

Comment:
I find this kind of post processing we have (turbo or not) very long. Would it make sense to break this up a bit? e.g. inlet quantities, outlet quantities, differences thereof.
Similarly these functions often run over a number of markers, maybe you could have an array of some type that stores the quantities per marker and the post processing function would operate on one marker at a time? (it would at least shorten the long lines)

Reply:
Have you done the same for Aerodynamic coefficients in your #861? But indeed the method can be improved and simplified. I ll definitely improved for the final PR to develop. For the timebeing I am just trying to have a common branch with @arubino so that he has something to work on for the new Turbo Output

Reply:
I only cleaned-up the mpi regions a little.